caom2pipe/execute_composable.py

```diff
@@ -1267,25 +1292,40 @@ def fix_compression(self, fqn):
 
 class FitsForCADCCompressor(FitsForCADCDecompressor):
     """
-    The class the implements the recompression.
+    This class implements conditional recompression.
+
+    SF - 20-05-22
```

Comment:
Is this used in the class?

Reply:
Yes - the code at line 1313 is the NOT of the `if` statement in the comments.